packages/adapter-cloudflare-workers/files/render.js

```diff
@@ -0,0 +1,70 @@
+import { render } from './app.js'; // eslint-disable-line import/no-unresolved
+import { getAssetFromKV, NotFoundError } from '@cloudflare/kv-asset-handler'; // eslint-disable-line import/no-unresolved
```

Comment:
any idea why the `eslint-disable-line` might be needed?

Reply:
yep. I don't particularly want to bundle up kv-asset-handler as part of the adapter, and the adapt function itself doesn't have a dependency on it, so lint complains.